src/schema/v2/partner/partnerInquiryRequest.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLList, GraphQLObjectType } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { InquiryQuestionType } from "../inquiry_question"
+import { LocationType } from "../location"
+import { InternalIDFields } from "../object_identification"
+import { InquirerCollectorProfileType } from "./partnerInquirerCollectorProfile"
+
+export const InquiryRequestType = new GraphQLObjectType<any, ResolverContext>({
+  name: "PartnerInquiryRequest",
+  fields: () => ({
+    ...InternalIDFields,
+    shippingLocation: {
+      type: LocationType,
+      resolve: ({ inquiry_shipping_location }) => inquiry_shipping_location,
+    },
+    questions: {
+      type: new GraphQLList(InquiryQuestionType),
+      resolve: ({ inquiry_questions }) => inquiry_questions,
+    },
```

Comment:
These are the two added fields; the rest was moved from the `partnerInquirerCollectorProfile.ts` file.